contracts/governance/InterchainMultisig.sol

```diff
@@ -0,0 +1,149 @@
+// SPDX-License-Identifier: MIT
+
```

Comment:
full unit test coverage is still missing, specifically some of the modifier checks (see branch columns etc.)
<img width="2550" alt="Screenshot 2024-02-20 at 6 26 33 PM" src="https://github.com/axelarnetwork/axelar-gmp-sdk-solidity/assets/14846413/6dd16d64-71ab-49c8-9e16-c61ff43c3b1a">

Reply:
still missing a few branches in coverage